Cargo.toml

```diff
@@ -41,8 +41,9 @@ mysql = { version = "20.0", optional = true }
 dashmap = { version = "4.0", optional = true }
 zip = { version = "0.5.8", optional = true }
 rand = {version = "0.8", optional = true}
-dmsort = {version = "1.0.0", optional = true}
+dmsort = {version = "1.0.0", optional = true }
```

Comment:
Also you might want to bump the version of the package itself
`version = "0.5.0"`

I guess with semver this would be `version = "0.6.0"`?

Reply:
why would you include the patch version

Reply:
> why would you include the patch version

I assumed that was standard with semver. RUSTG has always been 0.x.0